tests/Commands/CommandDispatchTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace JustBetter\MagentoWebhooks\Tests\Commands;
+
+use Illuminate\Support\Facades\Bus;
+use Illuminate\Testing\PendingCommand;
+use JustBetter\MagentoWebhooks\Commands\CleanLogsCommand;
+use JustBetter\MagentoWebhooks\Jobs\CleanLogsJob;
+use JustBetter\MagentoWebhooks\Tests\TestCase;
+
+class CommandDispatchTest extends TestCase
+{
+    /** @test */
+    public function test_it_dispatches_job(): void
```

Comment:
It's helpful to have more descriptive test cases as it currently doesn't tell that much. Also, the `test_` prefix is not required with the `@test` comment.